Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Specify the version of Go to use
+FROM golang:1.19
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
```

Comment:
It would be cool if there was simpler way to do this automatically multiarch

I am assuming that for gitHub the default is amd64 so this is fine. But something to be aware while running locally

Reply:
As a sandbox project, this is fine but we will need multi-arch builds and images. See dapr/dapr as reference.

Reply:
For now, we have no plans to use it locally, so its going to be used by the github actions runner only.